bukkit/src/main/resources/resources/bukkit/config.yml

```diff
@@ -36,6 +36,16 @@ options:
   trim-gui-size: true
   # Enable/disable titles
   titles-enabled: true
+  # Enable/disable BossBars
+  bossbar:
+    progress: true
+    complete: true
+    # Time in seconds of bossbar showed
+    time: 10
+  # Enable/disable ActionBar
+  actionbar:
+    progress: true
+    complete: true
```

Comment:
Can these be false by default

Reply:
I have also set this to 5 seconds on testing.

Reply:
I changed default config for 5 seconds